spacy/pipeline/entityruler.py

```diff
@@ -141,7 +142,9 @@ def __call__(self, doc: Doc) -> Doc:
 
     def match(self, doc: Doc):
         self._require_patterns()
-        matches = list(self.matcher(doc)) + list(self.phrase_matcher(doc))
+        with warnings.catch_warnings():
+            warnings.filterwarnings("ignore", message="\\[W036")
```

Comment:
You could also call `errors.filter_warning` which does the escaping for you, but it's still only a one-liner so either way's fine